lib/github.atd

```diff
@@ -1193,4 +1193,66 @@ type update_gist = {
   files: (string * update_gist) list <json repr="object">
 } <ocaml field_prefix="update_gist_">
 
+
 type emojis = (string * string) list <json repr="object">
+
+type timeline_action = [
+  | Assigned <json name="assigned">
+  | Closed <json name="closed">
+  | Commented <json name="commented">
+  | Committed <json name="committed">
+  | Cross_referenced <json name="cross-referenced">
```

Comment:
It's really neat that GitHub used dashes instead of underscores for this name and only this name.